images/macos/toolsets/toolcache-10.13.json

```diff
@@ -1,8 +1,5 @@
 {
     "@actions/toolcache-python-macos-1013-x64": [
```

Comment:
On High Sierra, this file is not used anymore. Please remove it fully

Reply:
Thank you, file is fully removed.